### PR TITLE
Update font-fira-mono-nerd-font from 2.2.1 to 2.2.2

### PR DESCRIPTION
**Pull request by ianrobrien**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by ianrobrien** [comment]
It looks like this version still will have the issue in https://github.com/Homebrew/homebrew-cask-fonts/issues/6679.

**Comment by miccal** [comment]
Ping @Finii.

**Comment by Finii** [comment]
@miccal Sorry... created PR manually to get this out of the way, based on output of automation stuff.

Creating the PR required to do some `brew` tests, but installing `brew` took an eternity, it clones two big repos?! Whatever. (Linux)

File: Casks/font-fira-mono-nerd-font.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask "font-fira-mono-nerd-font" do
-  version "2.2.1"
-  sha256 "e60a9f8cd097c2cbaa54dc08dc861d6d02809cfc0d35d60410648cefb1855c23"
+  version "2.2.2"
+  sha256 "29fff85e0afe0bd723fbd84e6c9587b08fd277cacc516d49bd379faa95612ac3"
 
   url "https://github.com/ryanoasis/nerd-fonts/releases/download/v#{version}/FiraMono.zip"
   name "FiraMono Nerd Font (Fira)"
```
Comment (Finii) [inline Casks/font-fira-mono-nerd-font.rb:6]: To be consistent with the names in other Nerd Font casks, this should be `Fura Mono` rather than `FiraMono`.

**Pull request closed without merging**